server/middleware/auth.js

```diff
@@ -11,18 +11,19 @@ const Auth = {
         if(token){
             jwt.verify(token, secret, (err, decoded) =>{
                 if(err){
-                    console.log(err);
                     return res.status(401).send({
                         message: 'You do not have permission to this page.'
                     });
                 }else{
                     // if everything is good, save to request for use in other routes
+                    //console.log(decoded)
                     req.decoded = decoded;
+                    //console.log(req.decoded.id)
                     next();
                 }
             });
         }else{
-            return res.status(401).send({ message: 'No token provided' })
+            return res.status(401).send({ message: 'User has either not Sign Up or Login.' })
```

Comment:
hmmm. don't think this error message is nice. It could be "You need to signup or login to perform this action"

Reply:
okay i have fixed that.